elements/rh-progress-stepper/demo/vertical-stepper.html

```diff
@@ -0,0 +1,41 @@
+<rh-progress-stepper orientation="vertical">
+  <rh-progress-step
+    state="active"
+    label="Active Step"
+    description="Currently working on this step"
+    aria-current="step">
+  </rh-progress-step>
+  
+  <rh-progress-step state="complete">
+    <a href="https://www.redhat.com" slot="label">Complete Step</a>
```

Comment:
how about `<rh-progress-step href="...">`?

Reply:
Love it! Will get that in.